docs/get-started/frameworks/nextjs.mdx

```diff
@@ -840,9 +827,9 @@ See [local images](https://nextjs.org/docs/pages/building-your-application/optim
 
 You might get this if you're using Yarn v2 or v3. See [Notes for Yarn v2 and v3 users](#notes-for-yarn-v2-and-v3-users) for more details.
 
-### What if I'm using the Vite builder?
+### Should I use the Vite or Webpack version?
 
-Storybook provides a Vite-based framework for Next.js. Follow the [installation instructions](#with-vite) and replace all instances of `@storybook/nextjs` with `@storybook/nextjs-vite`.
+We recommend using [`@storybook/nextjs-vite`](./nextjs-vite.mdx) (Vite-based) for most projects because it offers faster builds, better test support, and a simpler configuration. However, if your project has custom Webpack configurations that are incompatible with Vite, use this framework (`@storybook/nextjs` - Webpack 5) instead.
```

Comment:
Copy looks good overall but something feels off about `use this framework (`@storybook/nextjs` - Webpack 5) instead.`

We could maybe remove the bit in between parentheses so it flows better?